manager.spec.yaml

```diff
@@ -132,7 +133,7 @@ paths:
         - BasicAuth: []
       responses:
         200:
-          description: Report of IDs of affected systems for a given CVE.
+          description: Report of IDs of affected systems for a given CVE. Rule presence and rule key parameters are deprecated and will get obsolete.
```

Comment:
Can you please add: `deprecated: true` to `rule_presence` and `rule_key` instead of this comment?